source/CommonJobs/CommonJobs.Application.EvalForm/Commands/ChangeResponsibleCommand.cs

```diff
@@ -0,0 +1,84 @@
+﻿using CommonJobs.Application.Evaluations.EmployeeSearching;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class ChangeResponsibleCommand : Command
+    {
+
+        public string _evaluatedUser { get; set; }
+
+        public string _period { get; set; }
+
+        public string _newResponsible { get; set; }
+
+        public ChangeResponsibleCommand (string evaluatedUser, string period, string  newResponsible)
+        {
+            _evaluatedUser = evaluatedUser;
+            _period = period;
+            _newResponsible = newResponsible;
+        }
+
+        public override void Execute()
+        {
+            var evaluationId = EmployeeEvaluation.GenerateEvaluationId(_period, _evaluatedUser);
+            var evaluation = RavenSession.Load<EmployeeEvaluation>(evaluationId);
+
+            if (evaluation == null)
+            {
+                throw new ApplicationException(string.Format("Error: Evaluación inexistente: {0}.", evaluationId));
+            }
+
+            // Check that the new responsable exist
+            var responsible = RavenSession
```

Comment:
it seems to be the _new_ responsible, could you make variable name more explicit about it?